echo_test.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from server import response_ok
+
+
+@pytest.fixture(scope='function')
+def test_response():
+    assert response_ok(client) ==  """ returns a response containg an inital response line, header, body"""
+    response_ok = """HTTP/1.1 200 OK\r\n\
+    DATE: Sun, 21 Jul 2001 23:32:15 GTM\r\n\
+    SERVER: Python/2.7.6\r\n\
+    \r\n\
+    """
```

Comment:
I'm confused by the syntax here.  Can you explain to me what exactly you are trying to accomplish?